components/operator/project.garden.yaml

```diff
@@ -7,12 +8,10 @@ variables:
 
 environments:
   - name: default
-    providers:
-    - name: kubernetes
-      setupIngressController: false
 
 providers:
   - name: kubernetes
+    setupIngressController: false
```

Comment:
Can you use a variable please? To avoid breaking changes for those who use the default configuration. With an override for your use case inside a `garden.default.env`, it should be ok for your use case.

Reply:
For my use case I don't need it and would've remove it. Thought it might be a good idea to keep it in, because the current config file contained this option. Do you still want me to use a variable or can we simply remove it?

Reply:
Ok, got it! Thks.